nbgrader/docs/source/user_guide/release/ps1/problem1.ipynb

```diff
@@ -1,31 +1,5 @@
 {
  "cells": [
-  {
```

Comment:
It looks like there are some cells removed from this notebook---was this intentional?

Reply:
Not sure why that happened. I replaced ps1 notebooks with their original version from the main repository